diff --git a/CMakeLists.txt b/CMakeLists.txt
index ad0269e..fbd35c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -344,34 +344,34 @@ endif(EIGEN_BUILD_PKGCONFIG)
 
 add_subdirectory(Eigen)
 
-add_subdirectory(doc EXCLUDE_FROM_ALL)
+#add_subdirectory(doc EXCLUDE_FROM_ALL)
 
-include(EigenConfigureTesting)
+#include(EigenConfigureTesting)
 
 # fixme, not sure this line is still needed:
-enable_testing() # must be called from the root CMakeLists, see man page
+#enable_testing() # must be called from the root CMakeLists, see man page
 
 
-if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
-  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
-else()
-  add_subdirectory(test EXCLUDE_FROM_ALL)
-endif()
+#if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
+#  add_subdirectory(test) # can't do EXCLUDE_FROM_ALL here, breaks CTest
+#else()
+#  add_subdirectory(test EXCLUDE_FROM_ALL)
+#endif()
 
-if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
-  add_subdirectory(blas)
-  add_subdirectory(lapack)
-else()
-  add_subdirectory(blas EXCLUDE_FROM_ALL)
-  add_subdirectory(lapack EXCLUDE_FROM_ALL)
-endif()
+#if(EIGEN_LEAVE_TEST_IN_ALL_TARGET)
+#  add_subdirectory(blas)
+#  add_subdirectory(lapack)
+#else()
+#  add_subdirectory(blas EXCLUDE_FROM_ALL)
+#  add_subdirectory(lapack EXCLUDE_FROM_ALL)
+#endif()
 
-add_subdirectory(unsupported)
+#add_subdirectory(unsupported)
 
-add_subdirectory(demos EXCLUDE_FROM_ALL)
+#add_subdirectory(demos EXCLUDE_FROM_ALL)
 
 # must be after test and unsupported, for configuring buildtests.in
-add_subdirectory(scripts EXCLUDE_FROM_ALL)
+#add_subdirectory(scripts EXCLUDE_FROM_ALL)
 
 # TODO: consider also replacing EIGEN_BUILD_BTL by a custom target "make btl"?
 if(EIGEN_BUILD_BTL)
@@ -382,18 +382,18 @@ if(NOT WIN32)
   add_subdirectory(bench/spbench EXCLUDE_FROM_ALL)
 endif(NOT WIN32)
 
-configure_file(scripts/cdashtesting.cmake.in cdashtesting.cmake @ONLY)
+#configure_file(scripts/cdashtesting.cmake.in cdashtesting.cmake @ONLY)
 
-ei_testing_print_summary()
+#ei_testing_print_summary()
 
 message(STATUS "")
 message(STATUS "Configured Eigen ${EIGEN_VERSION_NUMBER}")
 message(STATUS "")
 
-option(EIGEN_FAILTEST "Enable failtests." OFF)
-if(EIGEN_FAILTEST)
-  add_subdirectory(failtest)
-endif()
+#option(EIGEN_FAILTEST "Enable failtests." OFF)
+#if(EIGEN_FAILTEST)
+#  add_subdirectory(failtest)
+#endif()
 
 string(TOLOWER "${CMAKE_GENERATOR}" cmake_generator_tolower)
 if(cmake_generator_tolower MATCHES "makefile")
